pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -209,15 +209,14 @@ def barrier(self, name: Optional[str] = None) -> None:
             rendezvous(name)
 
     def __transfer_distrib_spawn_state_on_fit_end(self, trainer: "pl.Trainer", results: Any) -> None:
+        rank_zero_debug("Transferring results back to main process ...")
```

Comment:
Tbh, I liked the `"cleaning up tpu spawn environment..."` warning/info message for TPU spawn haha.
So it usually takes a lot of time transferring the results to the main process, and it's a good indicator that this step has been completed.

No strong opinion tho

Reply:
which part of the implementation of this function is cleaning things up? As I see it, it is quite the opposite of cleaning up, as it creates a checkpoint file. 

Debug was requested by @carmocca and @tchaton 
I align with you but also I don't care so I will let you all decide

Reply:
> As it usually takes a lot of time transferring the results to the main process for TPUs

I'd cater to the most noob users. They don't know that this is slow, or what are "results" referring to, so I'd avoid the message.

But also no strong opinion :P

Reply:
I'd prefer info with an old message. At least during cleanup users won't think that it's stuck somewhere and will get the idea that cleanup is happening.

Reply:
Let's maybe use something more general as "Finalizing the TPU spawn environment. It shouldn't be long."

Reply:
I would say that finalizing sounds good... without any time estimete

Reply:
"Finalizing the TPU spawn environment."

Reply:
I would vote for debug level. We already have many things that are too verbose IMO.

Reply:
I never said there is a vote haha 🤣 But anyway, this is the current state:

warning: nobody
info: Kaushik, Adrian, Rohit,
debug: Carlos, Justus
unclear: Jirka, Thomas